nilearn/plotting/displays.py

```diff
@@ -1910,6 +1910,7 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
                                                self._cut_displayed)
         return cut_coords
 
+    @classmethod
```

Comment:
this should be `staticmethod` otherwise the bound method will have the class as its first parameter. btw it is a bit confusing that some class methods in this module call their first parameter `self` rather than the more conventional `cls`, but I guess that is outside the scope of this PR

Reply:
Ah, right, thank you.

Reply:
See #2923 for self -> cls.